github/src/main/kotlin/gropius/sync/github/model/IssueInfo.kt

```diff
@@ -31,13 +32,18 @@ data class IssueInfo(
     /**
      * Time of the last accessed timeline item
      */
-    var lastAccess: OffsetDateTime?
+    var lastAccess: OffsetDateTime?,
+    val imsProject: String
 ) {
     /**
      * MongoDB ID
      */
     @Id
     var id: ObjectId? = null
+
+    /**
+     * Turn the IssueInfo into a gropius Issue
```

Comment:
missing docs for param and return value

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105

Reply:
docs for return value still missing

Reply:
af217fde871ded5eb3e03a21b7d408b96b04189a